Revert commit a2790163af4d3f375a778055d0e2699207dfd050
and commit 60d719fe9105431a0d5361dbd54db20b6eba7234

--- origsrc/evolution-data-server-3.18.1/addressbook/libebook/Makefile.am	2015-03-23 01:41:04.000000000 -0500
+++ src/evolution-data-server-3.18.1/addressbook/libebook/Makefile.am	2015-11-06 11:56:59.107381400 -0600
@@ -59,7 +59,6 @@ libebook_1_2_la_LIBADD = \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/addressbook/libedata-book/libedata-book-1.2.la \
-	$(top_builddir)/private/libedbus-private.la \
 	$(EVOLUTION_ADDRESSBOOK_LIBS) \
 	$(CAMEL_LIBS) \
 	$(NULL)
--- origsrc/evolution-data-server-3.18.1/addressbook/libedata-book/Makefile.am	2015-03-23 01:41:04.000000000 -0500
+++ src/evolution-data-server-3.18.1/addressbook/libedata-book/Makefile.am	2015-11-06 11:57:23.175437700 -0600
@@ -51,7 +51,6 @@ libedata_book_1_2_la_LIBADD = \
 	$(top_builddir)/addressbook/libegdbus/libegdbus-book.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
-	$(top_builddir)/private/libedbus-private.la \
 	$(DB_LIBS) \
 	$(CAMEL_LIBS) \
 	$(SQLITE3_LIBS) \
@@ -128,7 +127,6 @@ evolution_addressbook_factory_subprocess
 evolution_addressbook_factory_subprocess_LDADD = \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
-	$(top_builddir)/private/libedbus-private.la \
 	libedata-book-1.2.la \
 	$(EVOLUTION_ADDRESSBOOK_LIBS) \
 	$(LIBSECRET_LIBS) \
--- origsrc/evolution-data-server-3.18.1/calendar/libedata-cal/Makefile.am	2015-03-23 01:41:04.000000000 -0500
+++ src/evolution-data-server-3.18.1/calendar/libedata-cal/Makefile.am	2015-11-06 11:58:17.708862500 -0600
@@ -40,7 +40,6 @@ libedata_cal_1_2_la_LIBADD = \
 	$(top_builddir)/calendar/libegdbus/libegdbus-cal.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
-	$(top_builddir)/private/libedbus-private.la \
 	$(EVOLUTION_CALENDAR_LIBS) \
 	$(CAMEL_LIBS) \
 	$(NULL)
@@ -100,7 +99,6 @@ evolution_calendar_factory_subprocess_SO
 evolution_calendar_factory_subprocess_LDADD = \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
-	$(top_builddir)/private/libedbus-private.la \
 	libedata-cal-1.2.la \
 	$(EVOLUTION_CALENDAR_LIBS) \
 	$(LIBSECRET_LIBS) \
--- origsrc/evolution-data-server-3.18.1/private/Makefile.am	2015-03-23 01:41:04.000000000 -0500
+++ src/evolution-data-server-3.18.1/private/Makefile.am	2015-11-06 10:25:23.870575000 -0600
@@ -200,7 +200,7 @@ BUILT_SOURCES = \
 	$(GENERATED_DBUS_SUBPROCESS_BACKEND) \
 	$(NULL)
 
-privsolib_LTLIBRARIES = libedbus-private.la
+noinst_LTLIBRARIES = libedbus-private.la
 
 # We omit AM_CPPFLAGS -- which is mostly warning flags -- since
 # we have no control over the C code generated by gdbus-codegen.
@@ -225,7 +225,6 @@ libedbus_private_la_LIBADD = \
 	$(NULL)
 
 libedbus_private_la_LDFLAGS = \
-	-avoid-version \
 	$(NO_UNDEFINED) \
 	$(CODE_COVERAGE_LDFLAGS) \
 	$(NULL)
